### DISCUSSION TITLE
Welcome to chartist Discussions!

### DISCUSSION THREAD

**Discussion by gionkunz** [chartist-js/chartist#1346] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hansmaad** [comment]
Omg, you're back to Chartist :) 
I'm still using Chartist in older projects. While I have to say that I switched to Chart.js in most newer projects, as I considered this here to be discontinued and Chart.js is not too bad. 
I'm looking forward to see a bright new future of this lib. I'm still looking for the perfect one :) Most chart libraries come with just too much bloat. Many more modern ones make use of ways to be more modular, but the core package is still huge. So, modernisation / modularisation is something that I'd look for in the future Chartist.
Happy to contribute... :-)

**Reply by gionkunz** [reply]
Yes, I'm planning to invest more time in the future again to implement long outstanding features and also add new ones based on new standard technology. @dangreen from cube invested a significant amount of time in renovating the stack and transitioning to TS. So a lot of credits need to go to them and Dan.
